odoo/addons/base/ir/ir_actions_report.py

```diff
@@ -85,7 +85,7 @@ def _merge_pdf(documents):
         for document in documents:
             pdfreport = open(document, 'rb')
             streams.append(pdfreport)
-            reader = PdfFileReader(pdfreport, overwriteWarnings=False)
+            reader = PdfFileReader(pdfreport, overwriteWarnings=False, strict=False)  # https://bugs.launchpad.net/pypdf/+bug/242755
```

Comment:
Fedex International deliver provider is giving extra-page in labels pdf. so we need to pass strict params with False value in PdfFileReader

@mep-odoo , can we add argument in _merge_pdf method: like strict=True
and pass strict=False from merge_pdf method of delivery object